docs/vlab/demo.md

```diff
@@ -7,7 +7,7 @@ You can find instructions on how to setup VLAB in the [Overview](overview.md) an
 
 ## Default topology
 
-The default topology is Spine-Leaf with 2 spines, 2 MCLAG leafs and 1 non-MCLAG leaf.
+The default topology is Spine-Leaf with 2 spines, 2 MCLAG leaves and 1 non-MCLAG leaf.
```

Comment:
unwanted change

Reply:
As discussed in Slack, I think it'll be easier and more consistent to allow "leafs" for now in the typo checker as CLI flags & output already contain a lot of it.

Reply:
added leafs as an extra word in the typos config